pipelines/tests/test_fit.sh

```diff
@@ -18,7 +18,7 @@ python pipelines/forecast_state.py \
 	--state-level-nssp-data-dir "$BASE_DIR/private_data/nssp_state_level_gold" \
 	--priors-path pipelines/priors/prod_priors.py \
 	--param-data-dir "$BASE_DIR/private_data/prod_param_estimates" \
-	--nwss-data-dir "$BASE_DIR/private_data/nwss_vintages" \
+	--nwss-data-dir "pipelines/tests/test_data/nwss_vintages" \
```

Comment:
Why not keep the data in the same place as before?

Reply:
I see. I have a proposal in https://github.com/CDCgov/pyrenew-hew/pull/413#discussion_r2029492448